code/api/src/modules/user/resolvers.js

```diff
@@ -64,6 +65,28 @@ export async function login(parentValue, { email, password }) {
   }
 }
 
+// Update
+export async function update(parentValue, { id, name, email, street, city, state, zip, bio, image }, { auth }) {
+  console.log(auth.user.id)
```

Comment:
NBD, but just pointing out this console.log is still here

Reply:
Good point @ rachel. I'll pull this down, remove it and push it back up to this PR